content/page/what.en.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "What"
-date: 2018-01-11 17:33:02
+date: 2020-07-23 17:45:00
 draft: false
 ---
 
-# What Exodus Privacy does
+# What εxodus Privacy does
```

Comment:
I know it's not always clear, but Exodus Privacy is the organization, εxodus the tool.
So there is no such thing as εxodus Privacy ;)

Reply:
Thanks, understood, i'll make the change.